xcodeproj/internal/opts.bzl

```diff
@@ -134,7 +138,7 @@ def _get_unprocessed_compiler_opts(*, ctx, target):
     cpp = ctx.fragments.cpp
 
     return (
-        base_copts + cpp.copts + cpp.conlyopts + user_copts,
+        base_copts + cpp.copts + cpp.conlyopts + user_copts + objc_copts,
```

Comment:
This needs to be before user ones, to make sure that user ones can override it. I'm not sure where in the order it is, but can you compare this to what Bazel produces? We might need to remove some of the existing ones as well depending on what you see.

Also, for the swift setting of the flags, I think we need to set a lot less. Looking at rules_swift and what they set, it's just a small subset with some filtered as well. We should mimic that.